packages/react-combobox/src/components/ComboButton/useComboButtonStyles.ts

```diff
@@ -7,11 +7,40 @@ export const comboButtonClassName = 'fui-ComboButton';
  * Styles for the root slot
  */
 const useStyles = makeStyles({
+  // TODO: add themed styles
   root: {
-    // TODO Add default styles for the root element
+    ...shorthands.border('1px', 'solid', '#d1d1d1'),
+    borderBottomColor: '#616161',
+    ...shorthands.borderRadius('4px'),
+    display: 'flex',
+    alignItems: 'center',
+
+    '&:focus-within': {
+      outlineWidth: '2px',
+      outlineStyle: 'solid',
+      outlineColor: 'transparent',
+      borderBottomColor: '#0078d4',
+      borderBottomWidth: '2px',
+    },
```

Comment:
let's be consistent how we style things

```suggestion
...createCustomFocusIndicatorStyle(
      {
        outlineWidth: '2px',
        outlineStyle: 'solid',
        outlineColor: 'transparent',
        borderBottomColor: '#0078d4',
        borderBottomWidth: '2px',
      },
      { selector: 'focus-within' },
    ),
  },
```

Reply:
This PR doesn't define the actual styles for Combobox, I just threw in some temporary stuff to give it enough of a working appearance that it can be tested (hard to check that arrow keys are working if the options don't highlight, for example 😅). I have a comment in there and a note in the PR, but I can make a bigger, more obvious comment if that would be safer.